### PR TITLE
Add vlans resource model for Junos OS

### PR DESCRIPTION
**Pull request by danielmellado**

### CODE REVIEW

File: models/junos/vlans/junos_vlans.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    state:
+      description:
+        - The state the configuration should be left in.
+      type: str
+      choices:
+        - merged
+        - replaced
+        - overridden
+        - deleted
+      default: merged
```
Comment (ganeshrn) [inline models/junos/vlans/junos_vlans.yml]: Please add these two sections in module doc
```
  requirements:
    - ncclient (>=v0.6.4)
  notes:
    - This module requires the netconf system service be enabled on
      the remote device being managed.
    - Tested against vSRX JUNOS version 18.4R1.
    - This module works with connection C(netconf). See L(the Junos OS Platform Options,../network/user_guide/platform_junos.html).
```

File: models/junos/vlans/overridden_example_01.txt [hunk 2]
```diff
@@ -0,0 +1,36 @@
+Using deleted
```
Comment (ganeshrn) [inline models/junos/vlans/overridden_example_01.txt]: Change to `overridden`

Comment (danielmellado) [reply models/junos/vlans/overridden_example_01.txt]: will change!

**Comment by danielmellado** [comment]
Hi Ganesh, actually regarding the state after checking I'll use that on the vlan_interface but not here, thanks!

**Comment by danielmellado** [comment]
Pull-request updated, HEAD is now b21a9a397fe55f928171f02140011ad3371c09b4

**Comment by danielmellado** [comment]
Pull-request updated, HEAD is now f64c93d79fd308447530d65ba6811e5c082a4806

**Pull request merged**